test/cli/test_cli_functions.sh

```diff
@@ -534,7 +534,7 @@ function noobaa_uninstall {
     [ ${check_cleanup_data_flag} -eq 0 ] && cleanup_data="--cleanup_data"
 
     echo_time "💬  Running uninstall ${cleanup} ${cleanup_data}"
-    test_noobaa --timeout uninstall ${cleanup} ${cleanup_data}
+    echo "y" | test_noobaa --timeout uninstall ${cleanup} ${cleanup_data}
```

Comment:
```suggestion
    yes | test_noobaa --timeout uninstall ${cleanup} ${cleanup_data}
```

Reply:
1. Would you like me to change the user input to `yes/no` (at the moment the input is `y/n`)?
2. Are you sure that the above is valid? I've created a 1 line script which calls `noobaa uninstall` to test this functionality, and without the `echo` it doesn't work.

Reply:
> 1. Would you like me to change the user input to `yes/no` (at the moment the input is `y/n`)?
I think `y/n` is good.
> 2. Are you sure that the above is valid? I've created a 1 line script which calls `noobaa uninstall` to test this functionality, and without the `echo` it doesn't work.
works for me: 
```
$ yes | build/_output/bin/noobaa-operator-local uninstall --cleanup
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n
INFO[0000] Will remove CRD (cluster scope)
INFO[0000] CLI version: 5.9.0
INFO[0000] noobaa-image: noobaa/noobaa-core:master-20210717
INFO[0000] operator-image: liranmauda/noobaa-operator:crt8347
INFO[0000] noobaa-db-image: centos/postgresql-12-centos7
INFO[0000] Namespace: default
INFO[0000]
INFO[0000] System Delete:
```

Reply:
Something is weird here.
You used `yes` which is not what the code expected (y/n), but for some reason instead of asking you again for input, it continued with the uninstall - Same happens to me (I don't see how the code is wrong though, any idea?).
Please try with `y` instead of `yes` - Without the echo it throws me in an endless loop:
```
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
INFO[0000] --cleanup removes the CRDs and affecting all noobaa instances, are you sure? y/n  
```
Can you confirm please?

Reply:
`yes` is a Linux command that prints a string repetitively, and the default is `y`, for reference:  [yes man page](https://man7.org/linux/man-pages/man1/yes.1.html)
What you are experiencing is looks like it prints something else the `y` and `n` 
Can you check the yes command output?

Reply:
Got it, thanks